codegenerator/cli/templates/dynamic/codegen/src/IO.res.hbs

```diff
@@ -139,9 +143,9 @@ let executeDbFunctionsEntity = (
   promises->Promise.all->Promise.thenResolve(_ => ())
 }
 
-let executeBatch = async (sql, ~inMemoryStore: InMemoryStore.t, ~isInReorgThreshold) => {
+let executeBatch = async (sql, ~inMemoryStore: InMemoryStore.t, ~isInReorgThreshold, ~config) => {
   let entityDbExecutionComposer =
-    RegisterHandlers.getConfig()->Config.shouldSaveHistory(~isInReorgThreshold)
```

Comment:
The problem with referencing the codegen config is that tests which pass in a different config have mixed states. I had this problem with a test that was using unordered multichain mode for some functions and not for others resulting in some weird behaviours